inquire-dir.f90

```diff
@@ -0,0 +1,8 @@
+! Result of INQUIRE on a directory
+! True: GNU, NVF, Cray
+! False: Intel
+! TBD: NAG, XLF, f18
+logical :: is_file
+inquire(file="foo", exist=is_file)
```

Comment:
`file="."` might be better, `.` is always a directory on Unix. Or `/`.

Reply:
Clever!  I've changed to `.`. and the results appear to be the same.  (Maybe `.` is better than `/` with respect to Windows testing...?  It's been a long time since I've used Windows.)